cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -892,70 +893,6 @@ declare namespace Deno {
     options: CreateHttpClientOptions,
   ): HttpClient;
 
-  /** **UNSTABLE**: New API, yet to be vetted.
-   *
-   * Represents membership of a IPv4 multicast group.
-   *
-   * @category Network
-   */
-  interface MulticastV4Membership {
-    /** Leaves the multicast group. */
-    leave: () => Promise<void>;
-    /** Sets the multicast loopback option. If enabled, multicast packets will be looped back to the local socket. */
-    setLoopback: (loopback: boolean) => Promise<void>;
-    /** Sets the time-to-live of outgoing multicast packets for this socket. */
-    setTTL: (ttl: number) => Promise<void>;
-  }
-
-  /** **UNSTABLE**: New API, yet to be vetted.
-   *
-   * Represents membership of a IPv6 multicast group.
-   *
-   * @category Network
-   */
-  interface MulticastV6Membership {
-    /** Leaves the multicast group. */
-    leave: () => Promise<void>;
-    /** Sets the multicast loopback option. If enabled, multicast packets will be looped back to the local socket. */
-    setLoopback: (loopback: boolean) => Promise<void>;
-  }
-
-  /** **UNSTABLE**: New API, yet to be vetted.
-   *
-   * A generic transport listener for message-oriented protocols.
-   *
-   * @category Network
-   */
-  export interface DatagramConn extends AsyncIterable<[Uint8Array, Addr]> {
-    /** Joins an IPv4 multicast group. */
-    joinMulticastV4(
-      address: string,
-      networkInterface: string,
-    ): Promise<MulticastV4Membership>;
-
-    /** Joins an IPv6 multicast group. */
-    joinMulticastV6(
-      address: string,
-      networkInterface: number,
-    ): Promise<MulticastV6Membership>;
```

Comment:
I wonder if this should be split out to a `UDPDatagramConn` as this `DatagramConn` interface would be useful to support datagrams-over-QUIC (which won't support multicast).

Reply:
I think it could just throw on those methods